content/3_Silver/Greedy_Sorting.problems.json

```diff
@@ -165,6 +165,18 @@
         "kind": "internal"
       }
     },
+    {
+      "uniqueId": "cf-1158A",
+      "name": "The Party and Sweets",
+      "url": "https://codeforces.com/problemset/problem/1158/A",
+      "source": "CF",
+      "difficulty": "Easy",
+      "isStarred": false,
+      "tags": ["2P", "Sorting", "Greedy"],
```

Comment:
Just Greedy and Sorting are sufficient here. You can also remove sorting if you want